tests/tst_storage.cpp

```diff
@@ -2013,11 +2013,10 @@ void tst_storage::tst_attendees()
     m_storage->save();
 
     reloadDb();
-    // Organizer is not in the attendee list but mKCal is adding him.
+
     const KCalendarCore::Incidence::Ptr refetched = m_calendar->incidence(event->uid());
     QVERIFY(refetched);
     QCOMPARE(refetched->organizer(), fetched->organizer());
-    QCOMPARE(refetched->attendees(), fetched->attendees() << KCalendarCore::Attendee(fetched->organizer().name(), fetched->organizer().email()));
```

Comment:
Maybe should keep the check that the attendee list does not contains Carl, but contains Alice and Bob with `QCOMPARE(refetched->attendees(), fetched->attendees());` ?